.github/workflows/ros2_build_test/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -l
+. /opt/ros/humble/setup.sh
+
+colcon build --continue-on-error --packages-skip alica_ros_proxy alica_tracing alica_ros_turtlesim alica_tests supplementary_tests alica_msgs
+
+# colcon test
+
+# colcon test-result --verbose
+
+
+if [ $? -eq 0 ]
```

Comment:
Note that when we change this, you have to retrieve or save $? immediately after the colcon build command, so uncommenting could cause problems.  Another choice is to do `colcon build <stuff> || exit 1`

Reply:
Modified it to that, it does make more sense that way